### PR TITLE
Feature/ add HTML skeleton

### LINKED ISSUE #11: Commit / push Specific Recipe view HTML skeleton
add to current PR

### PR DESCRIPTION
**Pull request by jrmedina**
What does this PR do?
 - This PR will add a HTML starting point (dashboard) to build off of at a later date 

Where should your reviewer start?
- Entire index.html

What (if any) features are you implementing?
- N/A

Is there any part of your codebase that you would like to have reviewed for possible improvements?
- N/A

What (if anything) did you refactor? Who is the contributor(s) of that code?
- Removed CSS background 

Were there any issues that arose?
- N/A

Is there anything that you need from your teammate(s)?
- N/A

Any other comments, questions, or concerns?
- Will need to add the "specific-card" section

### CODE REVIEW

**Comment by dstavis** [comment]
Pushed a commit to this branch that adds the specific recipe template (I made a couple class-name changes but not much else). Also added .hidden{ display:none} to the css file so that the sections we want to be hidden on the landing page will be hidden by default. Any concerns or suggestions about those changes? Think they should be part of this PR, or no?

File: src/styles.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.hidden {
+  display: none;
```
Comment (dstavis) [inline src/styles.css:2]: @jrmedina @sway3406

**Review by sway3406 — COMMENTED** [review]
Skeleton looks clean and has good starting classes and ids

File: src/styles.css (hunk 1 above)
Comment (sway3406) [reply src/styles.css:2]: Looks good, maybe we could add the red border property to all containers for styling too?

**Pull request merged**